src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Feedback, FeedbackType, MANUAL_ASSESSMENT_IDENTIFIER } from 'app/entities/feedback.model';
+import { cloneDeep } from 'lodash';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+    selector: 'jhi-code-editor-tutor-assessment-inline-feedback',
+    templateUrl: './code-editor-tutor-assessment-inline-feedback.component.html',
+})
+export class CodeEditorTutorAssessmentInlineFeedbackComponent {
+    MANUAL = FeedbackType.MANUAL;
+    @Input()
+    get feedback(): Feedback {
+        return this._feedback;
+    }
```

Comment:
Does this need to be a getter? I think this can be a standard input 🤔

Reply:
I wanted to intercept the input with a setter and this is how it is written down in the angular docu:
https://angular.io/guide/component-interaction#intercept-input-property-changes-with-a-setter
Just used it that way.